lib/superbolt/version.rb

```diff
@@ -1,3 +1,3 @@
 module Superbolt
-  VERSION = "0.7.5"
+  VERSION = "0.7.6"
```

Comment:
@ajstiles There a 0.7.6 in the queue already. Mind reviwing it? https://github.com/socialchorus/superbolt/pull/13 :-)